x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('EmptyEngineOverview', () => {
 
   it('renders correctly versioned documentation URLs', () => {
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      `https://www.elastic.co/guide/en/app-search/${CURRENT_MAJOR_VERSION}/index.html`
+      `${docLinksService.appSearchBase}/index.html`
```

Comment:
[not a blocker or change request, just me thinking out loud] this is an empty string though right, in reality? I wonder if we should bother trying to mock it for tests at some point